src/client/styles/variables/_colors.scss

```diff
@@ -1,17 +1,20 @@
 $green: #08ab6b;
 $light-green: #e1f5c4;
-$purple: #5562eb;
 $red: #fa0202;
 $red-light: #e0a5a5;
 $gray1: #646c73;
```

Comment:
Color $gray1: #646c73 (used to text-normal) is similar to rgb(65, 64, 66)/#414042; which is defined in Sandstream colors book. It's a subtle change, but maybe it's better to use it with a stencil?

Reply:
I've tried and it's much more darker. So for now I leave it as it is.